shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -171,7 +163,20 @@ - (BOOL)application:(UIApplication*)application
   if ([_lifeCycleDelegate application:application openURL:url options:options]) {
     return YES;
   }
-  return [self openURL:url];
+
+  __block BOOL openURLSuccess = NO;  // Flag to track success
```

Comment:
`// Flag to track success` is unnecessary here: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-useless-documentation

Reply:
thanks